docs/src/SUMMARY.md

```diff
@@ -39,10 +39,10 @@
 
 # Appendix
 
-* [Forge Commands](appendix/forge.md)
-    * [test](appendix/forge/test.md)
-    * [init](appendix/forge/init.md)
-    * [clean-cache](appendix/forge/clean-cache.md)
+* [`snforge` Commands](appendix/forge.md)
```

Comment:
```suggestion
* [`snforge` Commands](appendix/snforge.md)
```
this should be probably changed as well?

Reply:
there are `forge.md` and `snfroge.md` files if I see correctly in the src/appendix. is this intended?

Reply:
the other one was not needed